appengine-java-logging/src/test/java/com/google/apphosting/logging/JsonFormatterTest.java

```diff
@@ -85,19 +104,155 @@ public void messageIncludesStackTrace() throws Exception {
     BufferedReader reader = new BufferedReader(new StringReader(message));
     assertEquals("logger: message", reader.readLine());
     assertEquals("java.lang.Throwable: thrown", reader.readLine());
-    assertTrue(
-        reader
-            .readLine()
-            .startsWith("\tat " + getClass().getName() + ".messageIncludesStackTrace"));
+    assertTrue(reader.readLine()
+        .startsWith("\tat " + getClass().getName() + ".messageIncludesStackTrace"));
+  }
+
+  @Test
+  public void testCommonsLogging() {
+    final Instant now = Instant.now();
+    final String expectedThreadName = Thread.currentThread().getName();
+
+    // Generate Events from Commons Logging
+    new CommonsLoggingExample().run();
+
+    // Verify that event was captured by slf4j -> java.util.logging
+    String[][] expected = new String[][] {{"DEBUG", "A CommonsLogging Debug Event: "},
+        {"INFO", "A CommonsLogging Info Event: "}, {"WARNING", "A CommonsLogging Warn Event: "},
+        {"ERROR", "A CommonsLogging Error Event: "}};
+
+    List<String> events = capturingHandler.getEvents();
```

Comment:
Each logger has different levels as well, by the time you implement it this way you are back to juts as verbose as above.